scripts/migration/migrate_primary_institutions.py

```diff
@@ -0,0 +1,53 @@
+import sys
+import logging
+
+from modularodm import Q
+
+from framework.transactions.context import TokuTransaction
+
+from scripts import utils
+from website.app import init_app
+from website.models import Node, NodeLog
+
+logger = logging.getLogger(__name__)
+logging.basicConfig(level=logging.WARN)
+
+PRIMARY_INSTITUTION_CHANGED = 'primary_institution_changed'
+PRIMARY_INSTITUTION_REMOVED = 'primary_institution_removed'
+
+def migrate(dry_run=True):
+    added_logs = NodeLog.find(Q('action', 'eq', PRIMARY_INSTITUTION_CHANGED))
+    for log in added_logs:
+        logger.warn('Log with id <{}> being updated for affiliation added'.format(log._id))
+        log.action = NodeLog.AFFILIATED_INSTITUTION_ADDED
+        log.save()
+
+    removed_logs = NodeLog.find(Q('action', 'eq', PRIMARY_INSTITUTION_REMOVED))
+    for log in removed_logs:
+        logger.warn('Log with id <{}> being updated for affiliation removed'.format(log._id))
+        log.action = NodeLog.AFFILIATED_INSTITUTION_REMOVED
+        log.save()
+
+    nodes = Node.find(Q('primary_institution', 'ne', None))
```

Comment:
This might not work, as you're removing `primary_institution` from the model. Can still use pymongo, something like
```python
from framework.mongo import database

nodes = database['node'].find({'_primary_institution': {'$ne': None}})
```